components/char_collection/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Fuchsia Authors. All rights reserved.
```

Comment:
Are we keeping this comment? I think we do given our license.

Reply:
Perhaps we keep it until char_collection is fazed out?

Reply:
You usually don't need to keep license related file level comments, if the license is the same as project license.
Since we are deriving work from Fuchsia sources, and that license is different from what we have, we may have to keep what you have in the future.
This should be discussed in the meeting (maybe open issue, mark as discuss).